packages/adaptivecards-tools-sdk/rollup.config.js

```diff
@@ -8,6 +8,7 @@ import cjs from "@rollup/plugin-commonjs";
 import typescript from "rollup-plugin-typescript2";
 import pkg from "./package.json";
 import json from "@rollup/plugin-json";
+import generateDeclarations from "rollup-plugin-generate-declarations";
```

Comment:
I don't know much about this part, maybe you can ask Ning Tang to help review it since he is the owner of TeamsFx SDK.

Reply:
Thanks for the information.